presto-product-tests/bin/run_on_docker.sh

```diff
@@ -88,6 +87,9 @@ if [[ $# == 0 ]]; then
   usage "<product test arguments>"
 fi
 
+INVOCATION_PARAMS_SHA1=$(echo "$@" | sha1sum | awk '{print $1}')
```

Comment:
It would be nice to know which reports are from which execution. I use them locally.

Reply:
Maybe I could add environment name to the name of the reports? It's not unique on it's own, but maybe would be enough for local usage context.